src/crafting_gui.cpp

```diff
@@ -1344,7 +1348,10 @@ std::pair<Character *, const recipe *> select_crafter_and_crafting_recipe( int &
                           crafter ) - crafting_group.begin();
 
     // Get everyone's recipes
-    const recipe_subset &available_recipes = crafter->get_group_available_recipes();
+    // WTF? If called with dummy npc, we have to do this. Why? Why doesn't Character::get_group_available_recipes()
```

Comment:
Looking at the implementation, Does the dummy have the inventory accessible through `dummy->crafting_inventory()` ? I think it is very separate in `inventory_override`.